src/Status.php

```diff
@@ -58,7 +59,7 @@ public function majorOutage(): bool
             return false;
         }
 
-        $majorOutageRate = (int) config('cachet.major_outage_rate', 25);
+        $majorOutageRate = (int) app(AppSettings::class)->major_outage_threshold;
```

Comment:
Because we're using statically typed settings, we don't need to cast here now.

```suggestion
        $majorOutageRate = app(AppSettings::class)->major_outage_threshold;
```

Reply:
Aaa yes, thanks 🙂 . Updated.